apps/src/ailab/Ailab.js

```diff
@@ -51,6 +51,16 @@ Ailab.prototype.init = function(config) {
   config.wireframeShare = true;
   config.noHowItWorks = true;
 
+  // We don't want icons in instructions
+  config.skin.staticAvatar = null;
+  config.skin.smallStaticAvatar = null;
+  config.skin.failureAvatar = null;
+  config.skin.winAvatar = null;
+
+  // Provide a way for us to have top pane instructions disabled by default, but
+  // able to turn them on.
+  config.noInstructionsWhenCollapsed = true;
```

Comment:
I think most of the time we use this setting to control if you get the CSP or CSF style instructions

Reply:
I borrowed this from [WebLab.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/weblab/WebLab.js#L139-L141), since I was basing the appearance here on Web Lab.  But I can remove if it won't be useful.

Reply:
Worth noting that I'm not yet sure whether we'll be using short or long/markdown instructions.

Reply:
I think that noInstructionsWhenCollapsed is the right call here as it is the style used in CSP and CSD. You can write "short instructions" in this format but we don't call them short instructions. Short instructions are a concept only for CSF style instructions where we always show some form of instruction so you can collapse to a shorter instruction. I don't think you want to use CSF style instructions.